pkg/cmd/spec/generator.go

```diff
@@ -42,16 +43,21 @@ func GenerateSpecWithSpinner(o *generator.Options, project *projectstack.Project
 
 	spec, err := g.GenerateSpec(o, stack)
 	if err != nil {
-		if !o.NoPrompt && sp != nil {
+		if !o.NoStyle && sp != nil {
 			sp.Fail()
 		}
+
+		if o.NoStyle {
+			return nil, errors.Wrap(stripansi.Strip(err.Error()))
```

Comment:
please add a todo comment here to indicate that we will replace this with KCL no-style flags when it is supported

Reply:
OK, I've  added the comment : )